helm/cluster-cloud-director/values.yaml

```diff
@@ -22,18 +22,19 @@ connectivity:
     sshTrustedUserCAKeys:
       - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIM4cvZ01fLmO9cJbWUj7sfF+NhECgy+Cl0bazSrZX7sU vault-ca@vault.operations.giantswarm.io
 controlPlane:
-  diskSizeGB: 30
```

Comment:
This means we need to set this explicitly in our values from now on. Right?

Reply:
Yes, otherwise if the template's base disk is >30GB and it is not changed in the values it will break the OS disk's FS

Reply:
How does this work in e2e tests? I don't see it there.

Reply:
If not specified it will default to the size of the VM template's disk, meaning it won't change the size.
We can add it to the e2e tests for alignment.